src/Server/modules/USPlacesManager.js

```diff
@@ -0,0 +1,325 @@
+const dataClassesModule = require('../modules/DataClasses.js');
+const readCSVModule = require('../modules/ReadCSV.js');
+const writeCSVModule = require('../modules/WriteCSV.js');
+
+var allUSPlaces = [];
+
+function InitializeAllUSPlace(){
+    allUSPlaces = readCSVModule.LoadUSData();
+}// read allUSdata and put into array
+
+function AddUSData(county, state, date, tod) {
+    
+    if(allUSPlaces == undefined){
+        InitializeAllUSPlace();
+    }
+
+	var errormsg = 'no error';
+	if (tod == 1) {
+		for (var i = 0; i < allUSPlaces.length; ++i) {
+			if (county == allUSPlaces[i].county && state == allUSPlaces[i].state) {
+				for (var key in allUSPlaces[i].currentNumConfirmed) {
```

Comment:
currentNumConfirmed is not dictionary, but it is an integer. Maybe we want to use numConfirmed instead?